docs/units-and-global-variables.rst

```diff
@@ -85,11 +85,6 @@ Block and Transaction Properties
     consecutive blocks in the canonical chain.
 
 .. note::
-    If you want to implement access restrictions in library functions using
-    ``msg.sender``, you have to manually supply the value of
-    ``msg.sender`` as an argument.
```

Comment:
What was this about?

Reply:
This was from the times where we still used callcode.

Reply:
Oh ok, makes sense then :)